src/ui/components/navigation-links/NavigationLinks.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent, Fragment } from 'react';
+
+import Hidden from '@material-ui/core/Hidden';
+import MenuIcon from '@material-ui/icons/Menu';
+
+import NAVIGATION_LINKS from '../../global/navigation';
+import RouterLinks from '../router-links';
+
+// TODO: Implement router links inside a menu in mobile, triggered by menu button
+const Navigation: FunctionComponent = () => {
+  return (
+    <Fragment>
+      <Hidden smDown>
+        <RouterLinks links={NAVIGATION_LINKS}></RouterLinks>
+      </Hidden>
+      <Hidden mdUp>
```

Comment:
added the icon for mobile so the layout doesn't look broken, there needs to be some kind of menu triggered from it. will make separate trello task for it